src/gettext.mk

```diff
@@ -12,6 +12,7 @@ $(PKG)_URL_2    := https://ftpmirror.gnu.org/gettext/$($(PKG)_FILE)
 # native gettext isn't technically required, but downstream
 # cross-packages may need binaries and/or *.m4 files etc.
 $(PKG)_DEPS     := cc libiconv $(BUILD)~$(PKG)
+$(PKG)_PATCHES  := $(TOP_DIR)/src/gettext-1-fixes.patch
```

Comment:
This isn't necessary, it's only required if the usual logic of `pkg-[0-9]*.patch` isn't suitable.